common/djangoapps/microsite_configuration/templatetags/microsite.py

```diff
@@ -63,3 +63,14 @@ def microsite_css_overrides_file():
         return "<link href='{}' rel='stylesheet' type='text/css'>".format(static(file_path))
     else:
         return ""
+
+
+@register.filter
+def microsite_template_path(template_name):
+    """
+    Django template filter to apply template overriding to microsites.
+    The django_templates loader does not support the leading slash, therefore
+    it is stripped before returning.
+    """
+    template_name = microsite.get_template_path(template_name)
+    return template_name[1:] if template_name[0] == '/' else template_name
```

Comment:
Now there are only 2 known use cases of this tag: "navigation.html" and "footer.html", but eventually it could be used on variables as well and the risk would rise.